src/components/Account/SubscriptionCard.tsx

```diff
@@ -73,37 +27,54 @@ export function SubscriptionCard() {
           </Title>
           {data?.canceledAt ? (
             <SubscribeButton priceId={data?.price.id}>
-              <Button compact variant="outline" rightIcon={<IconRotateClockwise size={16} />}>
+              <Button compact radius="xl" rightIcon={<IconRotateClockwise size={16} />}>
                 Resume
               </Button>
             </SubscribeButton>
           ) : (
-            <ManageSubscriptionButton>
-              <Button compact variant="outline" rightIcon={<IconSettings size={16} />}>
-                Manage
-              </Button>
-            </ManageSubscriptionButton>
+            <Button
+              compact
+              radius="xl"
+              color="gray"
+              rightIcon={<IconSettings size={16} />}
+              component={NextLink}
+              href="/user/membership"
+            >
+              Manage
+            </Button>
           )}
         </Group>
         {isLoading ? (
           <Center p="xl">
             <Loader />
           </Center>
         ) : data ? (
-          <DescriptionTable items={details} />
+          <Group position="apart">
+            <Group>
+              {image && (
+                <Center>
+                  <EdgeMedia src={image} width={40} />
+                </Center>
+              )}
+              {product && <Text>{product.name}</Text>}
+            </Group>
+            <Stack spacing={0}>
+              {price && (
+                <Text>
+                  {getStripeCurrencyDisplay(price.unitAmount, price.currency) +
+                    ' ' +
+                    price.currency.toUpperCase() +
+                    '/' +
+                    shortenPlanInterval(price.interval)}
+                </Text>
+              )}
+              <Text size="sm" color="dimmed">
+                {data.cancelAtPeriodEnd ? 'Ends' : 'Renews'}{' '}
+                {new Date(data.currentPeriodEnd).toLocaleDateString()}
```

Comment:
Maybe use `formatDate` util?
```suggestion
                {formatDate(data.currentPeriodEnd)}
```